web/src/App.jsx

```
@@ -2302,11 +2467,11 @@ function App() {
                   <strong>Access PulseLink Web anytime:</strong> Visit web.pulselink.app from any browser to manage your contacts, view synced messages, customize themes, and track emergency locations. All settings sync automatically with your mobile app.
                 </div>
               </div>
-              <div className="home-grid">
-                <button className="home-card" onClick={() => setActivePanel('pulselink')}>
-                  <div className="home-icon pulselink">
-                    <img src={logo} alt="PulseLink" />
-                  </div>
+            <div className="home-grid">
+              <button className="home-card" onClick={() => setActivePanel('pulselink')}>
+                <div className="home-icon pulselink">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)